src/ExpressionEvaluator/Core/Source/ResultProvider/Portable/Resources.resx

```diff
@@ -117,6 +117,18 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="DynamicView" xml:space="preserve">
+    <value>Dynamic View</value>
+    <comment>IDynamicMetaObjectProvider and System.__ComObject expansion</comment>
+  </data>
+  <data name="DynamicViewNotDynamic" xml:space="preserve">
+    <value>Only COM or Dynamic objects can have Dynamic View</value>
```

Comment:
I admit it's kinda lame, but it's meant to distinguish "Dynamic" (__ComObject and IDMOP) from "dynamic" in C#.

FWIW, I just copied the error message from the old implementation here.